File: Python/1. starrectangle.py
'''Cretae a Pattern like this 

*****

*****

*****
'''


row = int(input("Enter No. of Rows: "))
col = int(input("Enter No. of Cols: "))

for i in range (0, row):
    for j in range(0, col):
        print("*",end="")
    print("\n")

File: C++/1. starrectangle.cpp
// Cretae a Pattern like this 

// *****

// *****

// *****



#include <iostream>
using namespace std;

int  main( ){
    int rowCount, colCount;

    cout<< "Enter number of rows: ";
    cin>> rowCount;

    cout<<"Enter number of cols: ";
    cin>> colCount;

    for (int row =0; row<rowCount; row++){
        for (int col = 0; col< colCount; col++){
            cout<<"* ";
        }
        cout<< "\n";
    }
    return 0;
}